Tools/gulp/gulp-regenerate-expected.js

```diff
@@ -93,6 +93,12 @@ function gulpRegenerateExpected(options, done) {
       args.push(opts.syncMethods);
     }
 
+    //package version
+    if(opts.pv){
```

Comment:
minor: needs a space between `)` and `{`. I'd recommend using Document Formatter in VS.